<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="158924-LCALOI" ignored="0" name="Demo" tests="2" failures="1" timestamp="10 Feb 2023 16:49:32 GMT" time="67.954" errors="0">
  <testcase name="DatHangKhoDC" time="35.797" classname="TestScript_Demo.DatHangFlows"/>
  <testcase name="DatHangKhoSieuThi" time="31.119" classname="TestScript_Demo.DatHangFlows">
    <failure type="java.lang.AssertionError" message="Case thất bại expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Case thất bại expected [true] but found [false]
at TestScript_Demo.DatHangFlows.DatHangKhoSieuThi(DatHangFlows.java:167)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- DatHangKhoSieuThi -->
</testsuite> <!-- Demo -->
